geth-patches/0006-latest-cht.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/light/postprocess.go b/light/postprocess.go
+index e7e51388..dc6562be 100644
+--- a/light/postprocess.go
++++ b/light/postprocess.go
+@@ -66,12 +66,20 @@ var (
+ 		chtRoot:       common.HexToHash("6f56dc61936752cc1f8c84b4addabdbe6a1c19693de3f21cb818362df2117f03"),
+ 		bloomTrieRoot: common.HexToHash("aca7d7c504d22737242effc3fdc604a762a0af9ced898036b5986c3a15220208"),
+ 	}
++
++	statusRopstenCheckpoint = trustedCheckpoint{
```

Comment:
I assume it has been calculated recently, right? When testing locally how long does it take to sync to the most recent block?

Reply:
Yes, I did it a couple of days ago and sync took 9-10 seconds on my laptop.